src/prpy/planning/chomp.py

```diff
@@ -84,7 +85,20 @@ def sync(self, robot):
                         body_name, os.path.basename(cache_path)
                     )
 
-                    self.module.computedistancefield(body, cache_filename=cache_path)
+                    # Temporarily disable other bodies for distance field computation
+                    other_bodies = []
+                    for other_body in self.env.GetBodies():
+                        if other_body == body:
+                            continue
+                        if not other_body.IsEnabled():
+                            continue
+                        other_bodies.append(other_body)
+                    other_savers = map(openravepy.KinBodyStateSaver, other_bodies)
```

Comment:
`Save_LinkEnable` is set by default [0], but I changed this anyways for clarity (-:

[0] http://openrave.org/docs/latest_stable/coreapihtml/classOpenRAVE_1_1KinBody_1_1KinBodyStateSaver.html#a97962441c40bb5e3ff13a385f7a84f75